build/scripts/build.ps1

```diff
@@ -273,6 +273,8 @@ function Deploy-VsixViaTool() {
         "Vsix\VisualStudioDiagnosticsWindow\Roslyn.VisualStudio.DiagnosticsWindow.vsix",
         "Vsix\VisualStudioIntegrationTestSetup\Microsoft.VisualStudio.IntegrationTest.Setup.vsix")
 
+    Get-Process -Name "devenv"
```

Comment:
Fixed. Updated to use `| Out-Host` as per our offline discussion (Write-Host doesn't print the useful information).